src/renderer/component/fileViewer/internal/player.jsx

```diff
@@ -11,7 +11,10 @@ import LoadingScreen from 'component/common/loading-screen';
 
 class MediaPlayer extends React.PureComponent {
   static MP3_CONTENT_TYPES = ['audio/mpeg3', 'audio/mpeg'];
+  static SANDBOX_TYPES = ['application/x-lbry', 'application/x-ext-lbry'];
```

Comment:
This is the most/only objectionable part for me. I would still strongly like to seek a way of triggering the behavior that does not require switching on media type. This is not me saying a custom media type is wrong; it is me saying that _only_ being able to trigger to the sandbox behavior via media type is wrong.

Please at least file/determine what metadata and sdk changes need to happen to facilitate this behavior without requiring this. @eukreign @lyoshenka

Reply:
There is a W3C proposal for something that seems very similar to `.lbry`, could we use that proposal instead of having our own?

Proposal: https://github.com/WICG/webpackage

The extension suggested in that proposal is `.wpk`:

https://github.com/WICG/webpackage/blob/master/draft-yasskin-dispatch-web-packaging.md#internet-media-type-registration

Reply:
A lot that makes sense, but it's a proposal 19 years in the making with zero adoption and no working tools.

I will reach out to the current maintainer.